app/stac_api/validators.py

```diff
@@ -0,0 +1,196 @@
+import logging
+import re
+from datetime import datetime
+
+from django.contrib.gis.geos import GEOSGeometry
+from django.core.exceptions import ValidationError
+from django.utils.translation import gettext_lazy as _
+from stac_api.utils import fromisoformat
+
+logger = logging.getLogger(__name__)
+
+MEDIA_TYPES = [
+    ('application/x.asc+zip', 'Zipped ASCII (.asc)', '.zip'),
+    ('application/x.asc-grid+zip', 'Zipped ASCII-GRID (.xyz)', '.zip'),
+    ('application/x.e00+zip 	Zipped', 'e00', '.zip'),
```

Comment:
Remove tab in media type

Reply:
They use `; ` (semi-colon + space) here: https://github.com/radiantearth/stac-spec/blob/v0.9.0/item-spec/item-spec.md#media-types.
So I'd suggests to stick to that, unless it causes problems with the choice field. In that case I'd drop the extra space and the world will still keep on turning ;-)

Reply:
P.S. and they call it `cloud-optimized` (with d)